Meta/CMake/Superbuild/CMakeLists.txt

```diff
@@ -1,16 +1,12 @@
-cmake_minimum_required(VERSION 3.16)
+cmake_minimum_required(VERSION 3.20)
 
 project(
     SerenitySuperbuild
     DESCRIPTION "Orchestrate host and target builds in a single build"
     LANGUAGES NONE
 )
 
-# NOTE: Before CMake 3.19, if a custom command is attached to multiple step targets for Makefile and Visual Studio generators,
-#       it might be run multiple times during the build. Enable new behavior of policy CMP0114 to avoid this.
```

Comment:
The comment seems appropriate to leave in place? just to say why we are manually setting this policy